### DISCUSSION TITLE
Return non-convergence flag of pipeflow

### DISCUSSION THREAD

**Discussion by osteinm** [e2nIEE/pandapipes#275] [Q&A] [answered]
Hi,

what would be a smart way to return a flag (apart from the printed one) that indicates if a pipeflow is not converged? Is there additionally a way of flagging a system that returns a physically incorrect (e.g. pressure below 0)?

Thank you for your support and all the best,
Oscar

**Comment by jkisse** [comment, marked as answer]
Hi Oscar,

after running a pipeflow, the parameter `net.converged` is set automatically set to True or False. If the pipeflow returns physically incorrect values, this is treated as "not converged", i.e. `net.converged = False`.

I hope this helps,
best regards
Jolando
